doc/Installation.md

```diff
@@ -0,0 +1,152 @@
+
+You have just created a wallet, now you need to unlock it (with the password we typed before) and from that moment you will be able to send and receive money from other wallets on the network.
+``` 
+/home/go/bin/tfchain wallet unlock
+Wallet password: 
+Unlocking the wallet. This may take several minutes...
+Wallet unlocked
+``` 
+
+Now you are free to play and explore the wallet subcommand of tfchainc, as before, the flag --help will guide you on all the options available.
+
+When I want to see all the transactions of our wallet 
+``` 
+/home/go/bin/tfchain transactions
+``` 
+
+There is a web based explorer that you can consult at [tfexplorer](http://185.69.166.13:2015/block.html?height=150)
+
+
+### Multiple Nodes ###
+
+The advantage of using docker is that we can create multiple nodes, just to play, or for fun or to explore the inners and working of tfchain.
+
+If you want to create a new node, you just ned to follow the very same steps outlined in the "Using docker" section but, you will need to change the name of the container, otherwise docker will complain and it will not work.
+
+``` 
+docker run -d --name tfchain2 --net tfnet -i -t ubuntu:16.04 /bin/bash
+``` 
+
+Once you have started the tfchaind in the second instance, it will recognice the first daemon running as the peers will comunicate the addres.
+
+If you don't see the address of your first tfchaind instance you can ask for a connection with it, where W.X.Y.Z is the addres of your first instance (btw: you may know the address of the node using the command "tfchain gateway address")
+``` 
+/home/go/bin/tfchain gateway connect W.X.Y.Z:23112
+``` 
+
+With docker you can create as much nodes as you wish or you workstation may stand.
+
+
+## In your workstation ##
+
+Most of the things donde to get, compile and run the tfchain on a docker container is applicable when you want to run the tfchain on your workstation.
+
+You need a golang development environment (version 1.9 or superior), usually Ubuntu has one version on its repositories, but also is an outdated one, for Ubuntu 16.04 the version supplied with the distribution is 1.6, and tfchain uses features of golang only available on 1.9. So you need to add a repository that contains the updated versions or you may also do a manual installation (as you can see it in [golang](http://golang.org/dl)).
+
+Once golang is up and running, you just need to set the GOROOT, GOPATH and PATH environment variables and from that moment, you just need to call go get
+``` 
+go get github.com/threefoldfoundation/tfchain/cmd/tfchaind 
+go get github.com/threefoldfoundation/tfchain/cmd/tfchainc
+``` 
+
+That will create the needed binaries and they will be placed in $GOPATH/bin. Now you can start the daemon **tfchaind** and use the client **tfchainc** to play with tfchain.
```

Comment:
You can/should remove the first sentence.

Please at the end link to the documentation explaining how to use those clients. If it doesn't exist yet, make an issue for it and work on that as well, while for now you already quickly could mention that they can use the `-h`/`--help`.